pages/interface/components/PublishedSince/index.js

```diff
@@ -14,9 +14,9 @@ function formatTooltipLabel(date) {
   return format(new Date(date), "EEEE, d 'de' MMMM 'de' yyyy 'às' HH:mm", { locale: ptBR });
 }
 
-export default function PublishedSince({ date }) {
+export default function PublishedSince({ direction, date }) {
   return (
-    <Tooltip sx={{ position: 'absolute', ml: 1 }} aria-label={formatTooltipLabel(date)}>
+    <Tooltip direction={direction} sx={{ position: 'absolute', ml: 1 }} aria-label={formatTooltipLabel(date)}>
```

Comment:
```suggestion
    <Tooltip sx={{ position: 'absolute', ml: 1 }} aria-label={formatTooltipLabel(date)} {...props}>
```

E qualquer propriedade extra que for passada para `PublishedSince` será repassada para o `Tooltip`.

Reply:
Perfeito!